casepro/contacts/tests.py

```diff
@@ -32,13 +32,15 @@ def test_save(self):
             language="eng",
             is_stub=False,
             fields={'age': "34"},
-            __data__groups=[("G-001", "Customers")]
+            __data__groups=[("G-001", "Customers")],
+            urns=["phone:0821234567"],
```

Comment:
Are we ok with using tel: for phone numbers? I appreciate thats a RapidPro thing but it's what we've been using up until now for outgoing messages.